src/ray/common/ray_config_def.h

```diff
@@ -307,7 +307,9 @@ RAY_CONFIG(uint32_t, cancellation_retry_ms, 2000)
 RAY_CONFIG(int64_t, ping_gcs_rpc_server_interval_milliseconds, 1000)
 
 /// Maximum number of times to retry ping gcs rpc server when gcs server restarts.
-RAY_CONFIG(int32_t, ping_gcs_rpc_server_max_retries, 600)
+/// ping_gcs_rpc_server_interval_milliseconds * ping_gcs_rpc_server_max_retries is
+/// recommended to be equivalent to gcs_server_request_timeout_seconds.
+RAY_CONFIG(int32_t, ping_gcs_rpc_server_max_retries, 60)
```

Comment:
Could we change the units to milliseconds and do the calculation internally? It's weird you have to do math to configure this properly.